### PR TITLE
Feature branch for review

### PR DESCRIPTION
**Pull request by SunnySparks**
### What this PR does 
- Add bubble_sort method
- Add bubble_sort file

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team. 👋

Nice work on this. 👌

However, there is just a slight issue:
Please fix it and submit another review request.

Refer to the inline code comments: 👇

HAPPY CODING ✨

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        array.insert(i + 1, deleted)
+      end
+    end
+  end
+  puts array
+end
+
+bubble_sort_by(%w[what is up yo]) do |left, right|
+  left.length - right.length
+end
```
Comment (mikenath223) [inline bubble_sort.rb]: -  [x] Please avoid using puts here and also delete the test you have here.

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      if array[i] > array[i + 1]
+        deleted = array.delete_at(i)
+        array.insert(i + 1, deleted)
+      end
+    end
+  end
+  puts array
+end
+
+bubble_sort([10, 30, 55, 98, 44, 31])
```
Comment (mikenath223) [inline bubble_sort.rb]: - [x] Kindly remove the puts on the array and remove the test case also.

Comment (Miguelus373) [reply bubble_sort.rb]: Thank you very much for your review

File: bubble_sort.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+def bubble_sort(array)
+  array.length.times do
+    (0...(array.length - 1)).each do |i|
+      if array[i] > array[i + 1]
+        deleted = array.delete_at(i)
+        array.insert(i + 1, deleted)
+      end
+    end
+  end
+  p array
```
Comment (kitpao) [inline bubble_sort.rb]: - [x] Please take out the 'p' and just leave `array` (do this in lines 10 and 24). The reason is that you are forcing the result to be printed, even when the user may not want to print it and just use the data, for example:
image
👉 2 printed results returned

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @SunnySparks and @Miguelus373 

I think there was a misunderstanding with the changes required by the previous TSE, please check below 🔽 

Note: Good work making your logic work since the beginning 👏 

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: bubble_sort.rb (hunk 3 above)
Comment (Miguelus373) [reply bubble_sort.rb]: Yes, it was a misunderstanding but is more clear now. Thank you!

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team ,  you have done a brilliant job with this project. :clap:
#### You can merge  !!!

**Pull request merged**